test-four/main.css

```diff
@@ -3,7 +3,34 @@ body {
 }
 
 .container {
-  padding: 30px;
+  padding: 2.1vw;
```

Comment:
can you explain why you're using a mixture of view width, pixels and rems?

Reply:
> Solid result, but would like to have seen a properly responsive solution.

Thanks! Yes in hindsight I probably should have used media queries I just wanted to try something different (If thats what you mean by properly responsive?).

Reply:
> can you explain why you're using a mixture of view width, pixels and rems?

I used view width here so when the window size changes the padding scales in a pleasing way. 

After some reading I can see that I should have used REMs for most things and used pixels a lot more sparingly(just for thin lines and some other measurements).